firestore-bigquery-export/firestore-bigquery-change-tracker/src/logs.ts

```diff
@@ -217,12 +217,12 @@ export const bigQueryTableInsertErrors = (
 ) => {
   logger.warn(`Error when inserting data to table.`);
 
-  insertErrors.forEach((error) => {
+  insertErrors?.forEach((error) => {
     logger.warn("ROW DATA JSON:");
     logger.warn(error.row);
 
     if (error && error.errors) {
-      error.errors.forEach((error) =>
+      error.errors?.forEach((error) =>
```

Comment:
Is the `If` check needed now that we have optional chaining?

Reply:
you're right, this can be simplified